statsmodels/kernel_methods/_kde1d_renormalization.py

```diff
@@ -162,25 +167,28 @@ def grid(self, N=None, cut=None, span=None):
         R = upper - lower
         kernel = self.kernel
 
-        # Compute the FFT with enough margin to avoid side effects
-        # here we assume that bw << est_R / 8 otherwise the FFT approximation is bad anyway
+        # Compute the FFT with enough margin to avoid side effects here we
+        # assume that bw << est_R / 8 otherwise the FFT approximation is bad
+        # anyway
         shift_N = N // 8
         comp_N = N + N // 4
         comp_lower = lower - R / 8
         comp_upper = upper + R / 8
 
         weights = self.weights
 
-        mesh, density = fftdensity(exog, kernel.rfft, bw, comp_lower, comp_upper, comp_N, weights, self.total_weights)
+        mesh, density = fftdensity(exog, kernel.rfft, bw, comp_lower,
+                                   comp_upper, comp_N, weights,
+                                   self.total_weights)
 
         mesh = mesh.full()
         mesh = mesh[shift_N:shift_N + N]
         density = density[shift_N:shift_N + N]
 
         # Apply renormalization
-        l = (mesh - lower) / bw
-        u = (mesh - upper) / bw
-        a1 = (kernel.cdf(l) - kernel.cdf(u))
+        lower = (mesh - lower) / bw
+        upper = (mesh - upper) / bw
```

Comment:
mistake here, this redefines `lower` and `upper` which are already in use.
use different names, e.g. low, upp, or `low_` `upp_`   
(I sometimes add trailing underline to signal it's temporary and similar to a name that is already in use)